CHANGE SingleItem, CategoryItem, AddToBasket - items without price ca…

cjohannsen opened the pull request:
…nnot longer be add to basket

### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been tested
- [x] Plugin can be built

@plentymarkets/ceres-io

cjohannsen committed:
CHANGE SingleItem, CategoryItem, AddToBasket - items without price cannot longer be add to basket

felixdausch commented on resources/js/src/app/components/item/SingleItem.js:
```diff
@@ -36,6 +37,18 @@ Vue.component("single-item", {
+                return !isNaN(defaultPrice.price.value);
+            }
+
+            return false;
```
Kannst du das nicht mit einer Zeile zusammenfassen:

```
return isDefined(this.currentVariation.prices.default) 
    && !isNaN(this.currentVariation.prices.default.price.value);
```

cjohannsen committed:
REFACTOR /workspace/localsystem/plugins/inbox/plugins/Ceres/resources/js/src/app/components/item/SingleItem.js
/workspace/localsystem/plugins/inbox/plugins/Ceres/resources/js/src/app/components/itemList/CategoryItem.js
/workspace/localsystem/plugins/inbox/plugins/Ceres/resources/js/src/app/filters/hasItemDefaultPrice.filter.js
/workspace/localsystem/plugins/inbox/plugins/Ceres/resources/views/Item/Components/SingleItem/SingleItem_Details.twig
/workspace/localsystem/plugins/inbox/plugins/Ceres/resources/views/ItemList/Components/CategoryItem.twig

felixdausch commented on resources/js/src/app/components/itemList/CategoryItem.js:
```diff
@@ -1,3 +1,5 @@
+import { isDefined } from "../../helper/utils";
```
Der Import wird nicht mehr benötigt

felixdausch commented on resources/js/src/app/components/item/SingleItem.js:
```diff
@@ -1,4 +1,5 @@
 import { transformVariationProperties } from "../../services/VariationPropertyService";
+import { isDefined } from "../../helper/utils";
```
Der Import wird nicht mehr benötigt

cjohannsen committed:
REFACTOR SingleItem.js, CategoryItem.js - remove unnecessary imports
